content/docs/studio/index.md

```diff
@@ -1,25 +1,26 @@
 # DVC Studio
 
 [`DVC Studio`](https://studio.iterative.ai/) is a comprehensive and interactive
-collaboration tool for your Machine Learning projects.
+web application to manage and collaborate on your Machine Learning projects
```

Comment:
why don't use the same simple language as we do in blog here:

```
DVC Studio is a web application that you can
[access online](https://studio.iterative.ai/) or even host on-prem. It works
with the data, metrics and hyperparameters that you add to your ML project
repositories.
```

Reply:
Changed.